backend/database-sqlite.js

```diff
@@ -3,88 +3,99 @@
 import { mkdir } from 'node:fs/promises'
 import { join, resolve } from 'node:path'
 import sqlite3 from 'sqlite3'
```

Comment:
In my PR #2737 I ended up having to replace `sqlite3` with `better-sqlite3` because `sqlite3` crashed when using workers. See 0635655eb9a6f16ce3749d98e99008138772ae82 for details.

My PR will have a merge conflict with yours, and a nice side-effect is that `better-sqlite3` has a simpler API. Would you considering making the switch here too?

Reply:
Sure, I'd rather avoid ending up with two separate SQLite dependencies